src/pages/policy/output/PolicyReproducibility.jsx

```diff
@@ -56,10 +53,78 @@ export default function PolicyReproducibility(props) {
           text="Copy"
           style={{ width: 100 }}
           onClick={() => {
-            navigator.clipboard.writeText(initialLines.join("\n"));
+            navigator.clipboard.writeText(codeLines.join("\n"));
           }}
         />
       </div>
     </>
   );
 }
+
+function getHeaderLines(metadata) {
+  return [
+    "from " + metadata.package + " import Microsimulation",
+    "from policyengine_core.reforms import Reform",
+    "from policyengine_core.periods import instant",
+    "import pandas as pd",
+    "",
+    "",
+  ];
+}
+
+function getBaselineDefinitionCode(metadata, policy) {
+  if (metadata.countryId !== "us" && metadata.countryId !== "enhanced_us") {
+    return [];
+  }
+
+  // Calculate the earliest start date and latest end date for
+  // the policies included in the simulation
+  let earliestStart = null;
+  let latestEnd = null;
+
+  for (const parameter of Object.keys(policy.reform.data)) {
+    for (const instant of Object.keys(policy.reform.data[parameter])) {
+      const [start, end] = instant.split(".");
+      if (!earliestStart || Date.parse(start) < Date.parse(earliestStart)) {
+        earliestStart = start;
+      }
+      if (!latestEnd || Date.parse(end) > Date.parse(latestEnd)) {
+        latestEnd = end;
+      }
+    }
+  }
+
+  return [
+    `"""`,
+    "In US simulations, use reported state income tax liabilities",
+    `"""`,
+    "def modify_baseline(parameters):",
+    `    parameters.simulation.reported_state_income_tax.update(start=instant("${earliestStart}"), stop=instant("${latestEnd}"), value=True)`,
+    "    return parameters",
+    "",
+    "class baseline_reform(Reform):",
+    "    def apply(self):",
+    "        self.modify_parameters(modify_baseline)",
+    "",
+    "",
+  ];
+}
+
+function getImplementationCode(metadata, timePeriod) {
+  const isCountryUS =
```

Comment:
also apply on line 76
```suggestion
  const US_REGIONS = ["us", "enhanced_us"];
  const isCountryUS = US_REGIONS.includes(metadata.countryId); 
```